features/tags_and_wrangling/tag_wrangling_fandoms.feature

```diff
@@ -52,39 +53,36 @@ Scenario: fandoms wrangling - syns, mergers, autocompletes, metatags
     And I press "Preview"
     And I press "Post"
   Then I should see "Work was successfully posted."
-  
+
   # editing non-canonical fandom in order to syn it to existing canonical merger
   When I follow "SG1"
     And I follow "Edit"
-    And I fill in "Synonym of" with "Stargate"
-  When "autocomplete tests with JavaScript" is fixed
-#    Then I should see "Stargate Atlantis" in the autocomplete
-#    And I should see "Stargate SG-1" in the autocomplete
+    And I enter "Stargate" in the "Synonym of" autocomplete field
+  Then I should see "Stargate Atlantis" in the autocomplete
+    And I should see "Stargate SG-1" in the autocomplete
   When I fill in "Synonym of" with "Stargate SG-1"
-    And I fill in "tag_media_string" with "TV"
-  When "autocomplete tests with JavaScript" is fixed
-#    And I should see "TV Shows" int the autocomplete
-    And I fill in "tag_media_string" with "TV Shows"
+  When I enter "TV" in the "tag_media_string_autocomplete" autocomplete field
```

Comment:
This should probably be an And, since it's right after a mother When. Also, the Then on the next line needs an indenting adjustment.